ansible/group_vars/all/vars

```diff
@@ -0,0 +1,23 @@
+# Default test env templates
+testenv_testbed_vmware_template: Template-testbed-201803050718
+testenv_controller_vmware_template: Template-CentOS-7.4
+
+# Testenv block definition
+provisioning:
+  name: "{{ testenv_name }}"
+  folder: "{{ testenv_vmware_folder }}"
+  portgroup_mgmt: "{{ testenv_mgmt_network }}"
+  portgroup_contrail: "{{ testenv_data_network }}"
+  vms:
+    controller:
+      template: "{{ testenv_controller_vmware_template }}"
+      nodes:
+      - name: "{{ testenv_name[:7] }}-controller"
```

Comment:
Perhaps it'd be better to just error early if `testenv_name` is longer than 7?

Reply:
Trigger an error in Jenkinsfile or add some logic to `vmware-deploy-testenv.yml` playbook?

Reply:
[Refinement] @sodar I strongly agree with @krdln, job should fail if ```testenv_name ``` is longer then 7 characters.

Reply:
If the error is in ansible but before anything side-effectful happens, I'm ok with that. I'd also check the machine name length instead of just testenv name, in case we change the naming scheme.

It'd be nice to also add a comment to param in deploy test and demo env jobs.